datasets/cn/national_congress/crawler.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+import csv
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+
+def crawl_item(input_dict: dict, context: Context):
+    name = input_dict.pop("姓名/Name")
+    ethnicity = input_dict.pop("民族/Ethnicity")
+    gender = input_dict.pop("性别/Gender")
+    delegation = input_dict.pop("代表团/Delegation")
+
+    entity = context.make("Person")
+    entity.id = context.make_slug(name, ethnicity, gender, delegation)
+
+    entity.add("name", name, lang="chi")
+    entity.add("gender", context.lookup("gender", gender))
```

Comment:
if you name the lookup `type.gender` then the context will use it automatically when adding the value

```suggestion
    entity.add("gender", gender)
```